pallets/space-ownership/src/lib.rs

```diff
@@ -17,7 +17,7 @@ pub mod pallet {
     use frame_system::pallet_prelude::*;
 
     #[pallet::config]
-    pub trait Config: frame_system::Config + pallet_spaces::Config {
+    pub trait Config: frame_system::Config + pallet_spaces::Config + pallet_profiles::Config {
```

Comment:
Can we loosely couple this using a trait? instead of tightly coupling it.

it will help during testing.

Reply:
Yeah, also will be great for contracts in future.